### PR TITLE
Update playgrounds to turbo

### PR DESCRIPTION
**Pull request by bidoubiwa**
Update playgrounds in order to be able to run them in hot loading with the packages:

- Updated: react, angular, vanilla and geosearch
- Removed: HTML as it was redundant with vanilla 
- Removed: vue as it was overcomplicated, it'll be added in a future PR

### CODE REVIEW

File: playgrounds/geo-javascript/src/app.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { instantMeiliSearch } from '../../../src/client/index'
+import { instantMeiliSearch } from '@meilisearch/instant-meilisearch'
```
Comment (brunoocasali) [inline playgrounds/geo-javascript/src/app.js:1]: This will use the local version and not the npm one, right?

Comment (bidoubiwa) [reply playgrounds/geo-javascript/src/app.js:1]: Yes! Local version :)

Comment (bidoubiwa) [reply playgrounds/geo-javascript/src/app.js:1]: https://turbo.build/repo/docs/handbook/workspaces#workspaces-which-depend-on-each-other

**Review by brunoocasali — APPROVED** [review]
👍

**Pull request merged**